examples/simple-examples/README.md

```diff
@@ -116,28 +116,31 @@ yarn run numbers:regions:list
+|                 | [./src/sms/groups/update/update.ts](./src/sms/groups/update/update.ts)                                     | `GROUP_ID`                                     |
+|                 | [./src/sms/groups/delete/delete.ts](./src/sms/groups/delete/delete.ts)                                     | `GROUP_ID`                                     |
+| Batches         | [./src/sms/batches/send.ts](./src/sms/batches/send.ts)                                                     | `PHONE_NUMBER` + `RECIPIENT_PHONE_NUMBER`      |
+|                 | [./src/sms/batches/sendTextMessage.ts](./src/sms/batches/sendTextMessage.ts)                               | `PHONE_NUMBER` + `RECIPIENT_PHONE_NUMBER`      |
+|                 | [./src/sms/batches/sendBinaryMessage.ts](./src/sms/batches/sendBinaryMessage.ts)                           | `PHONE_NUMBER` + `RECIPIENT_PHONE_NUMBER`      |
+|                 | [./src/sms/batches/sendMediaMessage.ts](./src/sms/batches/sendMediaMessage.ts)                             | `PHONE_NUMBER` + `RECIPIENT_PHONE_NUMBER`      |
+|                 | [./src/sms/batches/list.ts](./src/sms/batches/list.ts)                                                     |                                                |
+|                 | [./src/sms/batches/dry-run.ts](./src/sms/batches/dry-run.ts)                                               | `RECIPIENT_PHONE_NUMBER`                       |
+|                 | [./src/sms/batches/get.ts](./src/sms/batches/get.ts)                                                       | `BATCH_ID`                                     |
+|                 | [./src/sms/batches/update.ts](./src/sms/batches/update.ts)                                                 | `BATCH_ID`                                     |
+|                 | [./src/sms/batches/replace.ts](./src/sms/batches/replace.ts)                                               | `BATCH_ID`                                     |
+|                 | [./src/sms/batches/cancel.ts](./src/sms/batches/cancel.ts)                                                 | `BATCH_ID`                                     |
+|                 | [./src/sms/batches/delivery-feedback.ts](./src/sms/batches/delivery-feedback.ts)                           | `BATCH_ID`                                     |
+| DeliveryReports | [./src/sms/delivery-reports/list.ts](./src/sms/delivery-reports/list.ts)                                   |                                                |
+|                 | [./src/sms/delivery-reports/getByBatchId.ts](./src/sms/delivery-reports/getByBatchId.ts)                   | `BATCH_ID`                                     |
+|                 | [./src/sms/delivery-reports/getByPhoneNumber.ts](./src/sms/delivery-reports/getByPhoneNumber.ts)           | `BATCH_ID`, `RECIPIENT_PHONE_NUMBER`           |
```

Comment:
This point was missed previously but `DeliveryReportsApi` should define `getForNumber` instead of `getByPhoneNumber` (or we have to review cross SDKs naming)

Reply:
Ticket created: DEVEXP-352